packages/web3-eth/src/web3_eth.ts

```diff
@@ -988,19 +988,61 @@ export class Web3Eth extends Web3Context<Web3EthExecutionAPI, RegisteredSubscrip
 	 *   value: '0x1'
 	 * }
 	 *
-	 * const transactionHash = await web3.eth.sendTransaction(transaction);
-	 * console.log(transactionHash);
-	 * > 0xdf7756865c2056ce34c4eabe4eff42ad251a9f920a1c620c00b4ea0988731d3f
+	 * const transactionReceipt = await web3.eth.sendTransaction(transaction);
```

Comment:
For minor changes pls create issue so some one from web3 team can update it. Its for avoiding airdrop farming.